src/app/shared/components/template/components/text-bubble/text-bubble.component.scss

```diff
@@ -3,67 +3,112 @@
 // NOTE - when copying css border-image double-slash // misinterpreted so replace with / 0 /
 
 $imageSize: 64px;
-$bubble-background-color-3: var(--text-bubble-background-color-3, --ion-color-primary-200);
-$bubble-border-color-3: var(--text-bubble-border-color-3, --ion-color-primary-500);
-$bubble-background-color-4: var(--text-bubble-background-color-4, --ion-color-secondary-200);
-$bubble-border-color-4: var(--text-bubble-border-color-4, --ion-color-secondary-500);
-
-.container {
+$bubble-background-color-1: var(--text-bubble-background-color-1);
+$bubble-border-color-1: var(--text-bubble-border-color-1);
+$bubble-background-color-2: var(--text-bubble-background-color-2);
+$bubble-border-color-2: var(--text-bubble-border-color-2);
+$bubble-background-color-3: var(--text-bubble-background-color-3);
+$bubble-border-color-3: var(--text-bubble-border-color-3);
+$bubble-background-color-4: var(--text-bubble-background-color-4);
+$bubble-border-color-4: var(--text-bubble-border-color-4);
+$bubble-background-color-5: var(--text-bubble-background-color-5);
+$bubble-border-color-5: var(--text-bubble-border-color-5);
```

Comment:
nit(blocking): When setting these variables, we should provide a default value to fall back on.

You've included default values in the `default` theme (`src/theme/themes/default.scss`), which is legitimate and will effectively apply the defaults. However, I think it's best to also include them here too, as fallback values for the variable definition (see the diffs for the deleted lines that these replace). That way it's an even more fundamental default, which will be applied even if no theme is active, i.e. the `default` theme is not applied.